exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
+library(tidyverse)
+library(ggthemes)
+
+#Load malaria data Rstudio
+
+malaria_data <- read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/malaria.dat",header=TRUE)
+
+#View malaria data
+
+malaria_data
+
+#Assign time and cases as per the data
+
+time <- malaria_data$Time
+
+cases <- malaria_data$Cases
+
+rain <- malaria_data$Rain
+
+#plot malaria cases over time and labels
+
+ggplot(malaria_data, aes(x= time, y= cases)) +geom_point() +labs(title="Malaria Cases Over Time")
```

Comment:
Good. I see that you have learned how to use ggplot for creating a dot plot or a point plot. Well done!

My question is how does this answer Task 2 which asks you to show the trend in cases over time? Are dot plots the best way to show trend?